frontend/app/src/components/home/communities/edit/Edit.svelte

```diff
@@ -45,6 +45,7 @@
     let channelsValid = true;
     let detailsValid = true;
     let rulesValid = true;
+    let canMakePublic = !editing || client.hasOwnerRights(original.membership.role);
```

Comment:
What about for making private? It doesn't seem like we check for having ownership rights anywhere for that...

Reply:
As in, this should probably be called `canChangeVisibility`.

Reply:
Ahh ok the `canMakePrivate` is worked out inside `VisibilityControl.svelte`.